src/AppKernel.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Contao extension for PHPStan.
+ *
+ * (c) David Greminger
+ *
+ * @license MIT
+ */
+
+namespace Contao\PhpStan;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\NewsBundle\ContaoNewsBundle;
+use Doctrine\Bundle\DoctrineBundle\DoctrineBundle;
+use Knp\Bundle\MenuBundle\KnpMenuBundle;
+use Knp\Bundle\TimeBundle\KnpTimeBundle;
+use Psr\Log\NullLogger;
+use Scheb\TwoFactorBundle\SchebTwoFactorBundle;
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\MonologBundle\MonologBundle;
+use Symfony\Bundle\SecurityBundle\SecurityBundle;
+use Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle;
+use Symfony\Bundle\TwigBundle\TwigBundle;
+use Symfony\Cmf\Bundle\RoutingBundle\CmfRoutingBundle;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\HttpKernel\Kernel;
+
+class AppKernel extends Kernel
+{
+    /**
+     * @return array<Bundle>
+     */
+    public function registerBundles(): array
+    {
+        return [
+            new FrameworkBundle(),
+            new SecurityBundle(),
+            new TwigBundle(),
+            new MonologBundle(),
+            new SwiftmailerBundle(),
+            new DoctrineBundle(),
+            new SchebTwoFactorBundle(),
+            new KnpTimeBundle(),
+            new KnpMenuBundle(),
+            new CmfRoutingBundle(),
+            new ContaoCoreBundle(),
+            new ContaoNewsBundle(),
```

Comment:
I think we have added the news-bundle here because it is required in the AppKernel of the functional tests in the core-bundle, haven't we? Because if so, we can remove it again here now.

Reply:
Fixed in 541851a. Had to activate the `ContaoManagerBundle` instead.